nfabackend/src/main/java/com/msg/nfabackend/entities/nfaCatalog.java

```diff
@@ -1,17 +1,304 @@
 package com.msg.nfabackend.entities;
 
+import java.io.IOException;
+import java.io.StringWriter;
+
+import javax.persistence.AttributeConverter;
 import javax.persistence.Column;
+import javax.persistence.Convert;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.Table;
+import javax.persistence.Transient;
+
+import com.fasterxml.jackson.core.JsonGenerationException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 @Entity
 @Table(name ="nfa")
 public class nfaCatalog {
 	
-	public nfaCatalog() {}
+	public static class BlueprintConverter implements AttributeConverter<NfaCatalogBlueprint, String> {
+		@Override
+		public String convertToDatabaseColumn(NfaCatalogBlueprint attribute) {
+			
+			if (attribute == null) {
+				return null;
+			}
+			
+			try (StringWriter stringWriter = new StringWriter()) {
+				new ObjectMapper().writeValue(stringWriter, attribute);
+				return stringWriter.toString();
+			} catch (IOException e) {
+				throw new IllegalStateException("Obj-to-JSON-Converting failed", e);
+			}
+			
+		}
+
+		@Override
+		public NfaCatalogBlueprint convertToEntityAttribute(String dbData) {
+			try {
+				return dbData == null ? null 
+						: new ObjectMapper().readValue(dbData, NfaCatalogBlueprint.class);
+			} catch (IOException e) {
+				throw new IllegalStateException("JSON-to-Obj-Converting failed", e);
+			}
+		}
+	}
+	
+	public static class NfaCatalogBlueprint {
+		private BpPropertyTemplateNoConditionDe de;
+		private BpPropertyTemplateNoConditionEn en;
+		/**
+		 * @return the de
+		 */
+		public BpPropertyTemplateNoConditionDe getDe() {
+			return de;
+		}
+		/**
+		 * @param de the de to set
+		 */
+		public void setDe(BpPropertyTemplateNoConditionDe de) {
+			this.de = de;
+		}
+		/**
+		 * @return the en
+		 */
+		public BpPropertyTemplateNoConditionEn getEn() {
+			return en;
+		}
+		/**
+		 * @param en the en to set
+		 */
+		public void setEn(BpPropertyTemplateNoConditionEn en) {
+			this.en = en;
+		}
+	}
+	
+	public static class BpPropertyTemplateNoConditionDe {
```

Comment:
What is the difference between this and the BpPropertyTemplateNoConditionEn? Why not use one Class twice???